pandas/core/base.py

```diff
@@ -791,7 +791,7 @@ def max(self):
         Return the maximum value of the object.
 
         Return the maximum value of the object within the same type.
-        Remember you also can access to a Dataframe's index as Dataframe.index.
+        The Index of a DataFrame can be accessed as pandas.DataFrame.index.
 
```

Comment:
I would rewrite the last sentence to:
The DataFrame Index can be accessed using pandas.DataFrame.index.

Reply:
Thanks! :)